connectors/aws/aws-bedrock/src/main/java/io/camunda/connector/aws/bedrock/BedrockConnectorFunction.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.aws.bedrock;
+
+import io.camunda.connector.api.annotation.OutboundConnector;
+import io.camunda.connector.api.outbound.OutboundConnectorContext;
+import io.camunda.connector.api.outbound.OutboundConnectorFunction;
+import io.camunda.connector.aws.bedrock.core.BedrockExecutor;
+import io.camunda.connector.aws.bedrock.model.BedrockRequest;
+import io.camunda.connector.generator.java.annotation.ElementTemplate;
+
+@OutboundConnector(
+    name = "AWS BedRock",
+    inputVariables = {"authentication", "configuration", "action", "data"},
```

Comment:
not sure it's up to date. `data` doesn't exist anymore, and we have `invokeModel` and `converse`instead I guess

Reply:
@johnBgood I think `data` is coming from here generic base model.